### ISSUE TITLE
can clicking on a link in an email open in a browser?

### ISSUE THREAD

**Issue by detrout** [flathub/org.kde.kontact#7]
I tried clicking on a link in an email and got a "Choose Application" dialog.

Unsurprisingly Firefox outside of the flatpak container was not listed.

Is there anything more convenient than copy link and paste into your regular browser?. I'm pretty sure there's serious security issues if a contained application can just call xdg-open

**Comment by danvratil** [comment]
Yep, looks like we use KRun to open URLs from the message viewer, which does not support Flatpak Portals.

I'll see what can be done about this :)

**Comment by danvratil** [comment]
Reported upstream: https://bugs.kde.org/show_bug.cgi?id=399380

**Comment by detrout** [comment]
Looks like there's some progress on getting this to work.

I tried clicking on a link and got the "pick your application" dialog from kontact. My best guess was to try /usr/bin/xdg-open. 

That got me an error about not being able to talk to org.kde.kioexec.kdbus${something}

I added "--own-name=org.kde.kioexec.*", to finish-args and then when I tell kontact to use xdg-open I get a Gnome looking popup asking what application I want to use to handle the request. If i pick a browser there the link opens in my unsandboxed browser.
